acquia_lift.module

```diff
@@ -819,6 +819,10 @@ function acquia_lift_personalize_agent_presave($agent) {
     $lift_api = AcquiaLiftAPIClientFactory::create($account, $plugin_name);
     $agent->machine_name = $lift_api->ensureUniqueAgentName($agent->machine_name, PERSONALIZE_MACHINE_NAME_MAXLENGTH);
   }
+  // Save the Lift Web site name to the test agent as it will be used as a
+  // prefix to disambiguate goals and decision sets.
```

Comment:
Other than decision sets and goals, we may also want to mention personalizations/campaigns here.